### DISCUSSION TITLE
Grafana v9.3: No Windows packages available for this release

### DISCUSSION THREAD

**Discussion by baur** [grafana/grafana#59640] [General]
There will be no support for Windows from now on?

![image](https://user-images.githubusercontent.com/2898361/205034316-4de1c20b-f51a-476d-adc9-3ff599eb3659.png)

**Comment by baur** [comment]
solved

**grafana-9.3.1.windows-amd64.zip**

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
